modules/k8s-operator-crd-support/scripts/wait_for_gatekeeper.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+is_deployment_ready() {
+    kubectl --context "$1" -n "$2" get deploy "$3" &> /dev/null
+    export exit_code=$?
+    while [ ! " ${exit_code} " -eq 0 ]
+        do
+            sleep 5
+            echo -e "Waiting for deployment $3 in cluster $1 to be created..."
+            kubectl --context "$1" -n "$2" get deploy "$3" &> /dev/null
+            export exit_code=$?
+        done
+    echo -e "Deployment $3 in cluster $1 created."
```

Comment:
Can we not just do `kubectl wait --for=condition=available --timeout=600s deployment/foo `

Reply:
I tested a few times with `kubectl wait` and it did not work. There were other issues as well so it might not have been the most scientific testing strategy :) I can go back and test it if you think it is cleaner. 

I ended up leaving it in as the service one required a "custom" check.

Reply:
I think we should try for the native wait if possible, although not a blocker. Yeah unfortunately we cant do that for svc.